pkg/limayaml/validate.go

```diff
@@ -595,3 +595,37 @@ func warnExperimental(y *LimaYAML) {
 		logrus.Warn("`mountInotify` is experimental")
 	}
 }
+
+// ValidateYAMLAgainstLatestConfig validates the values between the latest YAML and the updated(New) YAML.
+func ValidateYAMLAgainstLatestConfig(yNew, yLatest []byte) error {
+	var l, n LimaYAML
+	var err error
+	if err = Unmarshal(yLatest, &l, "Unmarshal latest YAML bytes"); err != nil {
+		return err
+	}
+	if err = Unmarshal(yNew, &n, "Unmarshal new YAML byte"); err != nil {
```

Comment:
We can avoid `err` declaration:
```suggestion
	if err := Unmarshal(yLatest, &l, "Unmarshal latest YAML bytes"); err != nil {
		return err
	}
	if err := Unmarshal(yNew, &n, "Unmarshal new YAML byte"); err != nil {
```